docs/table-partitioning-bucketing-sort-order.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "Table partitioning, bucketing and sorting order"
+slug: /table-partitioning-bucketing-sort-order
+date: 2023-12-25
+keyword: Table Partition Bucket Distribute Sort By 
+license: Copyright 2023 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Partitioned table
+
+Currently, Gravitino supports the following partitioning strategies:
+
+:::note
+The `score`, `dt`, and `city` appearing in the table below refer to the field names in a table.
```

Comment:
The use of dt, which I assume means date time, could be a little confusing, try not to use abbreviations like this.
I also remove "in the table"